### ISSUE TITLE
load_cast_forecast nordpool fails Hello,

### ISSUE THREAD

**Issue by renaatdb** [davidusb-geek/emhass#309]

**Comment by JohanAlvedal** [comment]
Got it working here. 
[https://github.com/JohanAlvedal/homeautomation/tree/main/Emhass/Shell](url)

**Comment by renaatdb** [comment]
> https://github.com/JohanAlvedal/homeautomation/tree/main/Emhass/Shell

Thanks, 
I'm going to study it

**Comment by purcell-lab** [comment]
There is no doubt adding in  these additional forecasts with some json foo can get quite complex quickly.

It would be useful if we could provide a blueprint/ template for some of the standard suppliers; nordpool, Amber, octopus.  So the end user could just tick a check box, rather than having all this yaml fun.

The other consideration is each time you modify the shell_command or rest_command you need to restart HA, it would also reduce complexity if you could just reload a template/ script each time you needed to modify the payload.

**Comment by JohanAlvedal** [comment]
Absolutely, I agree with you. Someone talented who could create a blueprint so you can enter your electricity supplier solcast and the most relevant parts, I think it would help many users of emhass. It was/is the part that many ausers struggle with to get emhass up and running, because it is precisely that part that is extremely complex. When and how should all scripts be run and how should they be configured.

**Comment by renaatdb** [comment]
Thanks, I thought I was the only one having trouble with this.  It's amazing how Emhass was made. However, it is difficult for a beginner to get started and difficult to understand the manual. 
Hopefully I'll manage to get everything working in the future.
I want to purchase a battery in the future and see what a dynamic rate (with Nordpool) can teach me.

**Comment by renaatdb** [comment]
Hello,
This is also an interesting tool from Thermia .
[](https://classiconlinestorage.blob.core.windows.net/web-resources/SmartPrice_Info_and_FAQ_NL.pdf)]

**Comment by davidusb-geek** [comment]
Closing this specific issue.
Feel free to open another one if needed.
